docs/source/basics.rst

```diff
@@ -43,9 +43,7 @@ We can print our pyQuil program (``print(p)``) to see the equivalent Quil repres
     X 0
 
 
-This isn't going to be very useful to us without measurements. In pyQuil 2.0, we have to ``DECLARE`` a memory space
-to read measurement results, which we call "readout results" and abbreviate as ``ro``. With measurement, our whole program
-looks like this:
+This isn't going to be very useful to us without measurements. To declare memory and measure into it, write:
```

Comment:
`To declare memory and measure into it, write` -> `To declare memory and write measurement readout data into it,`